shell/src/test/java/alluxio/cli/fs/command/DuCommandTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fs.command;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import alluxio.AlluxioURI;
+import alluxio.SystemOutRule;
+import alluxio.client.file.URIStatus;
+import alluxio.wire.FileInfo;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class DuCommandTest {
```

Comment:
up to you but we have DuIntegrationTest if you want to consolidate

Reply:
I am aware but I specifically made this test because running a full alluxio cluster is unnecessary in order to test this functionality. The time it would require to run this in an integration test is probably 10x higher than using a unit test - and we should work to reduce the execution times of our tests because they have been jumping higher recently.